diff -Naur opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/CMakeLists.txt opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/CMakeLists.txt
--- opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/CMakeLists.txt	2017-12-16 01:15:14.000000000 +0100
+++ opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/CMakeLists.txt	2018-03-21 15:37:15.617106119 +0100
@@ -7,10 +7,10 @@
   if (NOT ${SIMULATOR_INSTALL_PREFIX} STREQUAL "")
     set(SIMULATOR_LIB_PATH ${SIMULATOR_INSTALL_PREFIX}/lib/companion${C9X_NAME_SUFFIX})
   else()
-    set(SIMULATOR_LIB_PATH ${CMAKE_INSTALL_PREFIX}/lib/companion${C9X_NAME_SUFFIX})
+    set(SIMULATOR_LIB_PATH ${LIB_INSTALL_DIR}/companion${C9X_NAME_SUFFIX})
   endif()
   message(STATUS "Simulators library search path: " ${SIMULATOR_LIB_PATH})
-  set(SIMULATOR_LIB_INSTALL_PATH ${CMAKE_INSTALL_PREFIX}/lib/companion${C9X_NAME_SUFFIX})
+  set(SIMULATOR_LIB_INSTALL_PATH ${LIB_INSTALL_DIR}/companion${C9X_NAME_SUFFIX})
 else()
   set(COMPANION_NAME "companion")
   set(SIMULATOR_NAME "simulator")
@@ -91,7 +91,7 @@
   endif()
 else()
   set(PTHREAD_LIBRARY pthread)
-  link_directories(/usr/local/lib)
+  link_directories(${LIB_INSTALL_DIR})
 endif()
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${WARNING_FLAGS}")
@@ -304,7 +304,7 @@
 qt5_wrap_cpp(companion_SRCS ${companion_MOC_HDRS})
 
 add_executable(${COMPANION_NAME} MACOSX_BUNDLE ${WIN_EXECUTABLE_TYPE} ${companion_SRCS} ${icon_RC})
-qt5_use_modules(${COMPANION_NAME} Core Widgets Network)
+qt5_use_modules(${COMPANION_NAME} Core Widgets Network Xml)
 target_link_libraries(${COMPANION_NAME} PRIVATE generaledit modeledit simulation ${CPN_COMMON_LIB} qcustomplot shared storage ${PTHREAD_LIBRARY} ${SDL_LIBRARY} ${WIN_LINK_LIBRARIES})
 
 PrintTargetReport("${COMPANION_NAME}")
diff -Naur opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/firmwares/er9x/er9xinterface.cpp opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/firmwares/er9x/er9xinterface.cpp
--- opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/firmwares/er9x/er9xinterface.cpp	2017-12-16 01:15:14.000000000 +0100
+++ opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/firmwares/er9x/er9xinterface.cpp	2018-03-21 13:48:04.323714492 +0100
@@ -79,39 +79,6 @@
   model.swashCollectiveSource = applyStickMode(model.swashCollectiveSource, mode);
 }
 
-#if 0
-unsigned long Er9xInterface::loadxml(RadioData &radioData, QDomDocument &doc)
-{
-  QDebug dbg = qDebug();
-  dbg.setAutoInsertSpaces(false);
-  dbg << "trying er9x xml import... ";
-
-  std::bitset<NUM_ERRORS> errors;
-
-  Er9xGeneral er9xGeneral;
-  memset(&er9xGeneral,0,sizeof(er9xGeneral));
-  if (!loadRadioSettingsDataXML(&doc, &er9xGeneral)) {
-    errors.set(UNKNOWN_ERROR);
-    return errors.to_ulong();
-  }
-  else {
-    radioData.generalSettings=er9xGeneral;
-    dbg << "version " << (unsigned int)er9xGeneral.myVers << " ";
-  }
-  for (int i=0; i<getCapability(Models); i++) {
-    Er9xModelData er9xModel;
-    memset(&er9xModel,0,sizeof(er9xModel));
-    if(loadModelDataXML(&doc, &er9xModel, i)) {
-      applyStickModeToModel(er9xModel, radioData.generalSettings.stickMode+1);
-      radioData.models[i] = er9xModel;
-    }
-  }
-  dbg << "ok";
-  errors.set(ALL_OK);
-  return errors.to_ulong();
-}
-#endif
-
 unsigned long Er9xInterface::load(RadioData &radioData, const uint8_t *eeprom, int size)
 {
   QDebug dbg = qDebug();
diff -Naur opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/firmwares/er9x/er9xinterface.h opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/firmwares/er9x/er9xinterface.h
--- opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/firmwares/er9x/er9xinterface.h	2017-12-16 01:15:14.000000000 +0100
+++ opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/firmwares/er9x/er9xinterface.h	2018-03-21 13:54:06.456493163 +0100
@@ -41,7 +41,6 @@
 
     virtual unsigned long loadBackup(RadioData &, const uint8_t * eeprom, int esize, int index);
 
-    virtual unsigned long loadxml(RadioData &radioData, QDomDocument &doc);
 
     virtual int save(uint8_t * eeprom, const RadioData & radioData, uint8_t version=0, uint32_t variant=0)
     {
diff -Naur opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/firmwares/ersky9x/ersky9xinterface.cpp opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/firmwares/ersky9x/ersky9xinterface.cpp
--- opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/firmwares/ersky9x/ersky9xinterface.cpp	2017-12-16 01:15:14.000000000 +0100
+++ opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/firmwares/ersky9x/ersky9xinterface.cpp	2018-03-21 13:48:20.573749434 +0100
@@ -23,7 +23,6 @@
 #include "rlefile.h"
 #include <iostream>
 #include <bitset>
-
 #define FILE_TYP_GENERAL 1
 #define FILE_TYP_MODEL   2
 
@@ -113,47 +112,6 @@
   model.swashCollectiveSource = applyStickMode(model.swashCollectiveSource, mode);
 }
 
-#if 0
-unsigned long Ersky9xInterface::loadxml(RadioData &radioData, QDomDocument &doc)
-{
-  QDebug dbg = qDebug();
-  dbg.setAutoInsertSpaces(false);
-  dbg << "trying ersky9x xml import... ";
-
-  std::bitset<NUM_ERRORS> errors;
-
-  Ersky9xGeneral ersky9xGeneral;
-  memset(&ersky9xGeneral,0,sizeof(ersky9xGeneral));
-  if(!loadRadioSettingsDataXML(&doc, &ersky9xGeneral)) {
-    errors.set(UNKNOWN_ERROR);
-    return errors.to_ulong();
-  }
-  else {
-    radioData.generalSettings=ersky9xGeneral;
-    dbg << "version " << (unsigned int)ersky9xGeneral.myVers << " ";
-  }
-  for(int i=0; i<getCapability(Models); i++) {
-    if (ersky9xGeneral.myVers == 10) {
-      if (!loadModelDataXML<Ersky9xModelData_v10>(&doc, &radioData.models[i], i, radioData.generalSettings.stickMode+1)) {
-        dbg << "ko";
-        errors.set(UNKNOWN_ERROR);
-        return errors.to_ulong();
-      }
-    }
-    else {
-      if (!loadModelDataXML<Ersky9xModelData_v11>(&doc, &radioData.models[i], i, radioData.generalSettings.stickMode+1)) {
-        dbg << "ko";
-        errors.set(UNKNOWN_ERROR);
-        return errors.to_ulong();
-      }
-    }
-  }
-  dbg << "ok";
-  errors.set(ALL_OK);
-  return errors.to_ulong();
-}
-#endif
-
 unsigned long Ersky9xInterface::load(RadioData &radioData, const uint8_t *eeprom, int size)
 {
   QDebug dbg = qDebug();
