diff -Naur opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/CMakeLists.txt opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/CMakeLists.txt
--- opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f.orig/companion/src/CMakeLists.txt	2017-12-16 01:15:14.000000000 +0100
+++ opentx-b55b68f191cfa22dee396ee2842e9fb836d9664f/companion/src/CMakeLists.txt	2018-03-21 15:37:15.617106119 +0100
@@ -7,10 +7,10 @@
   if (NOT ${SIMULATOR_INSTALL_PREFIX} STREQUAL "")
     set(SIMULATOR_LIB_PATH ${SIMULATOR_INSTALL_PREFIX}/lib/companion${C9X_NAME_SUFFIX})
   else()
-    set(SIMULATOR_LIB_PATH ${CMAKE_INSTALL_PREFIX}/lib/companion${C9X_NAME_SUFFIX})
+    set(SIMULATOR_LIB_PATH ${LIB_INSTALL_DIR}/companion${C9X_NAME_SUFFIX})
   endif()
   message(STATUS "Simulators library search path: " ${SIMULATOR_LIB_PATH})
-  set(SIMULATOR_LIB_INSTALL_PATH ${CMAKE_INSTALL_PREFIX}/lib/companion${C9X_NAME_SUFFIX})
+  set(SIMULATOR_LIB_INSTALL_PATH ${LIB_INSTALL_DIR}/companion${C9X_NAME_SUFFIX})
 else()
   set(COMPANION_NAME "companion")
   set(SIMULATOR_NAME "simulator")
@@ -91,7 +91,7 @@
   endif()
 else()
   set(PTHREAD_LIBRARY pthread)
-  link_directories(/usr/local/lib)
+  link_directories(${LIB_INSTALL_DIR})
 endif()
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${WARNING_FLAGS}")
